ci-operator/config/openshift/aws-vpce-operator/openshift-aws-vpce-operator-main.yaml

```diff
@@ -38,6 +54,29 @@ tests:
   commands: make test
   container:
     from: src
+- as: stage-e2e-harness
+  commands: |
+    export REPORT_DIR="$ARTIFACT_DIR"
+    export CONFIGS="rosa,stage,addon-suite"
```

Comment:
I think @MrSantamaria is renaming any occurence of "addon". Are these suites accurate for individual operator tests?